src/apps/patron-page/PatronPage.dev.tsx

```diff
@@ -1,21 +1,14 @@
 import React from "react";
 import { ComponentMeta, ComponentStory } from "@storybook/react";
-import { configTypes } from "../../core/utils/helpers/fetcher";
 import PatronPage from "./PatronPage.entry";
+import serviceUrlArgs from "../../core/storybook/serviceUrlArgs";
 
 export default {
   title: "Apps / Patron page",
   component: PatronPage,
   argTypes: {
+    ...serviceUrlArgs,
     // Config
-    [configTypes.fbs]: {
-      defaultValue: "",
-      control: { type: "text" }
-    },
-    [configTypes.publizon]: {
-      defaultValue: "",
-      control: { type: "text" }
-    },
```

Comment:
Consider exporting named args for each service.

I recognize the need to avoid repeated controls for each service but this pattern means that any app using a single url will export controls for all urls. This can make it harder to grok which services are needed for what apps.

Reply:
I am deliberately using all the service urls in all the apps to make sure that we do not miss one. It is rough, I know, but I think it will minimize future errors.